cmd/kube-apiserver/app/options/validation.go

```diff
@@ -49,6 +51,27 @@ func validateServiceNodePort(options *ServerRunOptions) []error {
 	return errors
 }
 
+func validateTokenRequest(options *ServerRunOptions) []error {
+	errors := []error{}
+
+	enableAttempted := options.ServiceAccountSigningKeyFile == "" ||
```

Comment:
am I reading this wrong or are all of these conditions backwards? (enable was attempted if a signing key file != "", right?)

Reply:
Ha, thanks. :)